src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(last-child) {
+  margin-right: 4px;
+}
```

Comment:
The `:not()` pseudo-class should contain a type selector or universal selector. This isn't valid: `:not(last-child)`. It should be `:not(:last-child)`.